src/handwritten/time.py

```diff
@@ -0,0 +1,68 @@
+import ctypes
+import functools
+from datetime import datetime, timezone
+
+@functools.total_ordering
+class AbsoluteTime(ctypes.Structure):
```

Comment:
On further reflection, I don't think users should actually interact with this type for now. Assuming that passes muster, where should it go? @bkeryan - thoughts?

Reply:
This is ctypes-specific, so it belongs in `_lib.py`.

I think the public API and interpreter interfaces should use the standard Python time types, NI's `hightime` package, or both:
- https://pypi.org/project/hightime/
- https://github.com/ni/hightime/tree/master

Note that `hightime` doesn't have the NI-BTF binary layout, so it will require conversion.

`nidaqmx.proto` uses `google.protobuf.Timestamp`, which already has a Python class with conversion methods.
https://github.com/protocolbuffers/protobuf/blob/main/python/google/protobuf/internal/well_known_types.py#L98

Reply:
Ooh, nice, thanks for the info! `hightime` is great, and since we have precedence with nimi-python, I'll shift to that. I'll still need this class for the lib version, and then something similar for gRPC to convert to `google.protobuf.Timestamp` to maintain at least nanosecond precision which that class supports. I'll add that, too.